src/Entity/EntityBrowser.php

```diff
@@ -188,6 +188,7 @@ public function getDisplay() {
    */
   protected function displayPluginCollection() {
     if (!$this->displayCollection) {
+      $this->display_configuration['instance_uuid'] = $this->uuid();
```

Comment:
I would like to avoid having any state information for entity browser instances in EntityBrowser class.